src/display-wayland.cc

```diff
@@ -1084,7 +1160,7 @@ struct window *window_create(struct wl_surface *surface, struct wl_shm *shm,
   window->rectangle.y = 0;
   window->rectangle.width = width;
   window->rectangle.height = height;
-  window->scale = 0;
+  window->scale = 1;
```

Comment:
I seemed to have had to change initial scale of window to 1 because I was otherwise getting:
```
wl_surface@3: error 0: Specified scale value (0) is not positive
```
, the conky window didn't show and the process didn't respond to `SIGINT`.

This only seems to have happened after `78ea7b69782e2be6b024d6f408d08d9b7ecd25d8` so I should check that out.

Reply:
I added the call to `wl_surface_set_buffer_scale` in this PR. It's not necessary... so I'll remove it.